spring-cloud-gcp-examples/spring-cloud-gcp-config-example/README.md

```diff
@@ -0,0 +1,83 @@
+# Spring Cloud Config Client for Google Cloud Runtime Configurator API
+
+This sample application demonstrates using the `Spring Cloud Config Client for GCP` in your code.
+
+## Setup
+1. Install the [Google Cloud SDK](https://cloud.google.com/sdk/). You
+  can do so with the following command:
+```
+  curl https://sdk.cloud.google.com | bash
+```
+1. Create, enable billing on a project on the [Google Cloud Console](https://console.cloud.google.com).
+1. Enable the [Runtime Config API](https://console.cloud.google.com/flows/enableapi?apiid=runtimeconfig.googleapis.com)
+1. [Set up authentication](https://cloud.google.com/authentication/docs) using a service account.
+
+## Testing
+
+1. Create a configuration using the [Google Cloud SDK](https://cloud.google.com/sdk/). The configuration name
+should be in the format `config`_`profile`, for example : `myapp_prod`
+```
+    gcloud beta runtime-config configs create myapp_prod
+```
+Then set the variables you wish to load.
+```
+  gcloud beta runtime-config configs variables set \
+    queue_size 25 \
+    --config-name myapp_prod
+  gcloud beta runtime-config configs variables set \
+    feature_x_enabled true \
+    --config-name myapp_prod
+```
+
+1. In your Spring Boot application directory ,
+add the following line to [src/main/resources/META-INF/spring.factories](src/main/resources/META-INF/spring.factories) :
+```
+  org.springframework.cloud.bootstrap.BootstrapConfiguration=org.springframework.cloud.gcp.config.GoogleConfigPropertySourceLocator
+```
+
+1. Add the following dependency to your [pom.xml](pom.xml):
+```
+  <dependency>
```

Comment:
Suggestion: you might want to follow the pattern of the Maven and Gradle snippet in the other READMEs (#102). If a user uses our BOM, they won't need the `version` either.